pages/styles/press.css

```diff
@@ -43,12 +43,8 @@
 }
 
 .customContent {
-  display: -ms-flexbox;
-  display: flex;
-  flex-flow: column;
-  justify-content: space-between;
+  -ms-flex-align: center;
```

Comment:
We've got autoprefixer working consistently in this repo, so we shouldn't need this 👍